csv2rdf/get_relations.py

```diff
@@ -0,0 +1,30 @@
+"""
+This script takes as many as input parameters as needed.
+All the parameters will be the CSV files that need to be converted 
+and merged into one single RDF file.
+For all the CSV files, this script extracts their headers and write them
+to a JSON file, which is used as the mapping file for the 
+csv2rdf_single_subject.py script.
+"""
+import csv
+import json
+import sys
+import os
+
+filenames = sys.argv[1:]
+output_name = "mapping.json"
+dt = {}
+dt["entity_type"] = []
+
+for filename in filenames:
+    with open(os.path.abspath(filename), "r") as csv_file:
```

Comment:
Why do you need the abspath call here?

Reply:
If we move the test files elsewhere (a test folder inside this directory) and do not use the abspath call, then Python is not able to find the file.